sqlite3.go

```diff
@@ -1492,6 +1507,25 @@ func (d *SQLiteDriver) Open(dsn string) (driver.Conn, error) {
 		C.sqlite3_close_v2(db)
 		return nil, err
 	}
+	
+	// _pragma_key
+    if encryptionKey != "" {
+        query := fmt.Sprintf("PRAGMA key = \"%s\";", pragmaKey)
```

Comment:
You can use simply `%q` for readability instead of `\"%s\"`.
See https://pkg.go.dev/fmt#hdr-Printing